README.md

```diff
@@ -56,7 +56,15 @@ By default it will try to communicate with LND using local gRPC connection and `
 You can use `--userest` to start using REST API (default is as mentioned gRPC) - in that case `--rpcserver` can be complete URL to the endpoint.
 
 Of course you can also start the agent on a remote node and just point it to the correct endpoint, but since this is not the default
-use cases flags `--userest` and `--rpcserver` are not advertied in help.
+use cases flags `--userest` and `--rpcserver` are not listed in help.
+
+For CoreLightning it will use UNIX domain socket. By default agent will try to use `~/.lightning/bitcoin/lightning-rpc`.
```

Comment:
split into two parts and create sections with an actual examples on how to run it for cln and how to run it for lnd, dont just explain things in long form, provide people something they can copy paste immediately and where its clearly shown how different setups can be configured

Reply:
And `Advanced options` section in text to go through them, changed the basic examples to be copy-pastable.